test_vm/Vagrantfile

```diff
@@ -34,7 +46,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         "password" => {
           "postgres" => "postgres"
         },
-        "version" => "9.3",
+        "version" => "9.4",
```

Comment:
> I think postgresql upgrade would also need the customized cookbook reflected in Cheffile, Cheffile.lock? Maybe we can get these through a separate PR.

Done in #422. I'll merge so that we can rebase and concentrate here on the disk resizing issue.